eventmesh-storage-plugin/eventmesh-storage-standalone/src/main/java/org/apache/eventmesh/storage/standalone/broker/MessageQueue.java

```diff
@@ -24,24 +24,27 @@
 
 import com.google.common.base.Preconditions;
 
+import lombok.Getter;
+
 /**
- * This is a block queue, can get entity by offset. The queue is a FIFO data structure.
+ *   This is a block queue, can get entity by offset. The queue is a FIFO data structure
```

Comment:
It seems meaningless to remove the last period from the original comment and add a space before it. CC @xwm1992